gatorgrade/main.py

```diff
@@ -16,6 +19,9 @@ def gatorgrade(
     """Run the GatorGrader checks in the gatorgrade.yml file."""
     # check if ctx.subcommand is none
     if ctx.invoked_subcommand is None:
+        checks = parse_config(FILE)
+        checks = run_commands_and_return_results(FILE)
+        checks = generate_config(FILE)
```

Comment:
The Generate team's main function should be called when the `gatorgrade generate` command is run. So, this function will need to be moved.